### PR TITLE
Revert "Gerrit client ensure authenticated at startup"

### PR DESCRIPTION
**Pull request by chaodaiG**
This reverts commit f9a75c2122363bf47eda2462beacfc069b1a9115.

Our gerrit prow instance failed to query after this, reverting for now

### CODE REVIEW

**Review by cjwagner — APPROVED** [review]

File: prow/gerrit/adapter/adapter.go [hunk 1]
```diff
@@ -91,54 +91,38 @@ func NewController(ctx context.Context, prowJobClient prowv1.ProwJobInterface, o
...
 			// No need to spin constantly, give it a break. It's ok that config change has one second delay.
 			time.Sleep(time.Second)
 		}
 	}()
-}
-
-func (c *Controller) applyGlobalConfigOnce(cfg config.Getter, gerritClient *client.Client, lastSyncTracker *syncTime, cookiefilePath, tokenPathOverride string) {
-	orgReposConfig := cfg().Gerrit.OrgReposConfig
-	if orgReposConfig == nil {
-		return
```
Comment (chaodaiG) [inline prow/gerrit/adapter/adapter.go:125]: The root cause was that it returned here before calling the first Authenticate. Will fix in a following PR

Comment (cjwagner) [reply prow/gerrit/adapter/adapter.go:125]: :man_facepalming:

**Pull request merged**